mappings/net/minecraft/entity/passive/VillagerEntity.mapping

```diff
@@ -8,11 +8,12 @@ CLASS net/minecraft/class_1646 net/minecraft/entity/passive/VillagerEntity
 	FIELD field_18534 gossip Lnet/minecraft/class_4136;
 	FIELD field_18535 gossipStartTime J
 	FIELD field_18536 experience I
-	FIELD field_18537 lastRestock J
+	FIELD field_18537 lastRestockTime J
 	FIELD field_18538 MEMORY_MODULES Lcom/google/common/collect/ImmutableList;
 	FIELD field_18539 SENSORS Lcom/google/common/collect/ImmutableList;
 	FIELD field_18851 POINTS_OF_INTEREST Ljava/util/Map;
-	FIELD field_19357 lastGossipDecay J
+	FIELD field_19357 lastGossipDecayTime J
+	FIELD field_19427 todayRestock I
```

Comment:
What is this?

Reply:
Look at **RestocksToday** in https://minecraft.gamepedia.com/Villager/ED

Reply:
`restocksToday` is clear that it's the number of restocks today. `todayRestock` doesn't make sense.